tools/image-info

```diff
@@ -2682,6 +2682,7 @@ def append_partitions(report, image):
                     part_device,
                     info,
                     devices_map[part_device],  # retrieves the associated Device Object
+                    None,  # Just a filesystem, no partitions on this device
```

Comment:
This looks fine, if you want it to work without a force merge this probably helps (untested but should work):
```python
diff --git a/tools/image-info b/tools/image-info
index da2f8d3..d2feafd 100755
--- a/tools/image-info
+++ b/tools/image-info
@@ -2678,13 +2678,20 @@ def append_partitions(report, image):
                 jsonschema.validate(options, info.get_schema()["properties"]["options"])
 
                 # Finally mount
-                mmgr.mount(mounts.Mount(
-                    part_device,
-                    info,
-                    devices_map[part_device],  # retrieves the associated Device Object
-                    part_mountpoint,
-                    options))
-
+                mnt_kwargs = {
+                    "name": part_device,
+                    "info": info,
+                    # retrieves the associated Device Object
+                    "device": devices_map[part_device],
+                    "target": part_mountpoint,
+                    "options": options,
+                }
+                # XXX: remove inspect and just add once osbuild PR#1501 is merged
+                import inspect
+                if "partition" in inspect.getfullargspec(mounts.Mount):
+                    # Just a filesystem, no partitions on this device
+                    mnt_kwargs["partition"] = None
+                mmgr.mount(mounts.Mount(**mnt_kwargs))
             if not root_tree:
                 raise RuntimeError("The root filesystem tree is not mounted")
 
```

Reply:
Updated the code to match your suggestion!